subiquity/server/controllers/timezone.py

```diff
@@ -19,20 +19,27 @@
 from subiquity.common.apidef import API
 from subiquity.common.types import TimeZoneInfo
 from subiquity.server.controller import SubiquityController
+from shutil import which
+import os
 
 log = logging.getLogger('subiquity.server.controllers.timezone')
 
+TIMEDATECTLCMD = which('timedatectl')
```

Comment:
I guess this sort of thing is a good example of why we shouldn't do things at import time...

Reply:
Done, the entire test about a working timedatectl and systemd enabled has been moved to a function called in the if statement.